includes/class-linkbacks-handler.php

```diff
@@ -423,11 +427,21 @@ public static function get_comment_link( $link, $comment, $args ) {
 	 * @return string the replaced/parsed author url or the original comment link
 	 */
 	public static function get_comment_author_url( $url, $id, $comment ) {
-		if ( $author_url = get_comment_meta( $id, 'semantic_linkbacks_author_url', true ) ) {
-			return $author_url;
+		$author_url = get_comment_meta( $id, 'semantic_linkbacks_author_url', true );
+		if ( ! $author_url ) {
+			return $url;
 		}
-
-		return $url;
+		if ( get_comment_meta( $id, 'webmention_source_url' ) ) {
```

Comment:
I am not sure if this is a good place... we should not change data in a "getter" function...